docs/servers/add-azure-server.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Add a Server to Azure
+intro: Devopness allows you to provision a server on Azure Resource Manager (RM) and manage it through Devopness.
+links:
+    overview:
+    quickstart:
+    previous: /docs/servers/add-server
+    next: /docs/servers/check-server-provision-logs
+    guides:
+    related:
+    featured:
+---
+
+> If you don't have an Azure credential, please follow the guide {% mentionPost "/docs/credentials/add-azure-credential" %}
+
+1. On Devopness, navigate to a project then select an environment
+1. If there's no other server in the selected environment, just click `ADD SERVER`
+1. If a list of resource cards is displayed, find the `Servers` card and click View on it
+1. On the upper-right corner of the list click `ADD SERVER`
+1. Choose `Microsoft Azure` as cloud provider
+1. Select a `Credential`
+    > If no credential is listed or you want to use a different one, click `Create a new credential` and follow the guide {% mentionPost "/docs/credentials/add-credential" %}
```

Comment:
The option label for "Create a new " is capitalized

```suggestion
    > If no credential is listed or you want to use a different one, click `Create a new Credential` and follow the guide {% mentionPost "/docs/credentials/add-credential" %}
```

![image](https://github.com/user-attachments/assets/d720b9d0-1267-4847-973f-f7a3e27e8958)